### DISCUSSION TITLE
Changing logo strange behaviour with GRUB

### DISCUSSION THREAD

**Discussion by EnricoTirri** [BartoszCichecki/LenovoLegionToolkit#1060] [Q&A] [answered]
Change boot logo works during bios boot, but makes logo disappear during windows boot.
I've a Lenovo Legion Pro 5 with dual boot managed through GRUB, if I use default logo I got Legion logo on both boots (so I've thought that changing one would have changed both).
There is a reason why of this strange behaviour? Is a problem of dual boot manager? Has it always supposed to work like that?

**Comment by BartoszCichecki** [comment, marked as answer]
It might be an issue with GRUB. I only tested this with Windows boot manager since Legions only ship with this OS.
